include/deal.II/grid/grid_generator.h

```diff
@@ -945,6 +945,49 @@ namespace GridGenerator
               const unsigned int  n_cells  = 0,
               bool                colorize = false);
 
+  /**
+   * Produce an eccentric hyper-shell, the region between two spheres centered
+   * on two distinct center points. One has to specify the <tt>inner_center</tt>
+   * and <tt>outer_center</tt>, with given <tt>inner_radius</tt> and
+   * <tt>outer_radius</tt>. The number <tt>n_cells</tt> indicates the number of
+   * cells of the resulting triangulation, i.e., how many cells form the ring
+   * (in 2d) or the shell (in 3d).
+   *
+   * By default, the outer boundary has the indicator 1 while the inner boundary
+   * has id 0. In 3d, this applies to both the faces and the edges of these
+   * boundaries.
+   *
+   * A SphericalManifold is attached to the outer boundary with an id of 1 while
+   * another SphericalManifold is attached to the inner boundary with an id of
+   * 0. A TransfiniteInterpolationManifold is attached to all other cells and
+   * faces of the triangulation with an id of 2.
+   *
+   * In 2d, the number <tt>n_cells</tt> of elements for this initial
+   * triangulation can be chosen arbitrarily. If the number of initial cells
+   * is zero (as is the default), then it is computed adaptively such that the
+   * resulting elements have the least aspect ratio.
+   *
+   * In 3d, only certain numbers are allowed, 6 (or the default 0) for a
+   * surface based on a hexahedron (i.e. 6 panels on the inner sphere extruded
+   * in radial direction to form 6 cells), 12 for the rhombic dodecahedron,
+   * and 96. This choice dates from an older version of deal.II before the
+   * Manifold classes were implemented: today all three choices are roughly
+   * equivalent (after performing global refinement, of course).
```

Comment:
I think it would be clearer (and future-compatible) to instead write that `n_cells` has the same meaning in this function as it does in `hyper_shell`.

Reply:
This comment and the comment above should be resolved by my last commit, but I don't know how to bring the updated versions of the .cc and .h files in here!